cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs

```diff
@@ -1018,20 +1018,20 @@ instance ToJSON (Alonzo.CollectError StandardCrypto) where
         object
           [ "kind" .= String "PlutusTranslationError"
           , "error" .= case err of
-              Alonzo.ByronInputInContext -> String "Byron input in the presence of a plutus script"
-              Alonzo.ByronOutputInContext -> String "Byron output in the presence of a plutus script"
-              Alonzo.TranslationLogicErrorInput -> String "Logic error translating inputs"
+              Alonzo.ByronTxOutInContext txOutSource ->
+                String $ "Byron output in the presence of a plutus script, output source: " <> show txOutSource
```

Comment:
What does this mean? A script is trying to witness a byron tx out?

Reply:
This is a failure to create a transaction context for Plutus, due to the presence of a Byron address in the transaction. It can manifest as either a newly created transaction output, or as an input being spent that is connection to an output in the UTxO that has a Byron address (the case is clarified by `txOutSource`)

Reply:
How about "Cannot construct a Plutus ScriptContext from this transaction due to the a Byron UTxO being created or spent: " <> show ...